CHANGELOG.md

```diff
@@ -84,7 +84,7 @@
 
 * Added CipherSuite API for specifying underlying primitives
 * Added support for specifying a slow password hashing function
-* Collapsed SignalKeyPair to X25519KeyPair
+* Collapsed SignalKeyPair to Curve25519KeyPair
```

Comment:
This can probably not be changed since it is historical.

Reply:
Yeah, that was a mistake.